_source/_posts/2019-09-19-nodejs-typescript.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: blog_post
+title: "Build a NodeJS App with TypeScript"
+author: ivo-katunaric
+description: "Learn to build a simple blog server using Node.js and TypeScript"
+tags: [nodejs, typescript, nestjs, secure, oauth, oidc, javascript]
+tweets:
+- "Learn to build a blog application using Node.js and TypeScript with NestJS! →"
+- "Yes, you can use TypeScript with Node.js! Here's an example of building a blog application using Node.js and TypeScript with NestJS! →"
+- "Can you use TypeScript with Node.js? Absolutely! Here's one approach to building a blog application with TypeScript and NestJS! →"
+image: blog/featured/okta-node-bottle-headphones.jpg
+---
+
+As dynamically typed languages became prominent during the last decade, typeless (or should I say lawless?) programming became the norm for the backend as well as the frontend.
+
+Many people believe the simplicity of "just writing" code is efficient for providing a proof of concept or prototyping applications.
+
+However, as those applications grow, the typeless code used to build them often becomes incredibly convoluted and more difficult (some would say impossible) to manage.
```

Comment:
This intro seems like it's "punching down" at JavaScript and might be a turnoff. Could invoke similar reactions we had from our Kotlin intro yesterday.